Dockerfile

```diff
@@ -40,6 +40,13 @@ RUN groupadd -r geodata -g 2500 && \
 
 FROM sphinxsearch_geodata
 
+# Define the build argument with a default value
+ARG VERSION="unknown"
+# Make the argument available for use in the Dockerfile
+ENV VERSION=${VERSION}
+# Create the directory if it doesn't exist and save the value of the VERSION argument to /usr/local/share/app/version.txt
+RUN mkdir -p /usr/local/share/app && echo "${VERSION}" > /usr/local/share/app/version.txt && chown -R geodata:geodata /usr/local/share/app
```

Comment:
You should here also sets the labels on the image like in `service-stac` for example, see https://github.com/geoadmin/service-stac/blob/1d0a9a7119761ffa677f132c3ab0013b4bf7ada0/Dockerfile#L151 
This allow to check the version without running the container it also add usefull informations of the build like the git hash and so on.